text/en/chapters/data-representation.md

```diff
@@ -682,6 +682,38 @@ The following interactive can be used to upload your own image, and experiment w
 
 {comment end}
 
+## Representing program instructions with bits
+In a similar fashion to representing text or numbers using binary, we can represent an entire actual program using binary.
```

Comment:
Add note here to recommend reading sections of the Programming Languages chapter.